OpenOversight/tests/routes/test_notes.py

```diff
@@ -280,7 +280,7 @@ def test_acs_can_delete_their_notes_in_their_department(mockdata, client, sessio
             officer_id=officer.id,
             created_by=user.id,
             created_at=datetime.now(),
-            updated_at=datetime.now(),
+            last_updated_at=datetime.now(),
```

Comment:
It's smöl, but it may be worth it to create a variable for `datetime.now()` so we aren't calling it twice.